IPython/display.py

```diff
@@ -1,16 +1,44 @@
 """Public API for display tools in IPython.
 """
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 #       Copyright (C) 2012 The IPython Development Team
 #
 #  Distributed under the terms of the BSD License.  The full license is in
 #  the file COPYING, distributed as part of this software.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Imports
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 
-from IPython.core.display import *
```

Comment:
I think it should be fine to leave this one. The point of this approach was to keep old code like this from breaking.

Reply:
The problem is is was triggering the deprecation warning at import time, which then would be silenced for users, as the default is `warning:once`, hence why I'm explicit.